tests/fixtures/sample_lint_data.py

```
@@ -0,0 +1,439 @@
+            "linter": "ansible-lint"
+        },
+        {
+            "file_path": "tasks/main.yml",
+            "line": 3,
+            "column": 1,
+            "rule_id": "risky-shell-pipe",
+            "message": "Shells that use pipes should set the pipefail option",
+            "severity": "MEDIUM",
+            "linter": "ansible-lint"
+        }
+    ]
```

Ruff (0.12.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)